packages/backend/src/ee/controllers/snowflakeController.ts

```diff
@@ -1,4 +1,8 @@
-import { ApiErrorPayload, ApiSuccessEmpty } from '@lightdash/common';
```

Comment:
moved controller to /ee,
There is no service for snowflake auth, so nothing else needs to be moved for now.